3-networks-dual-svpc/modules/hierarchical_firewall_policy/versions.tf

```diff
@@ -19,11 +19,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.50"
+      version = ">= 3.82"
```

Comment:
Thanks - Can you also update for hub-and-spoke?

Reply:
sure, my bad